src/openms/source/ANALYSIS/QUANTITATION/TMTTenPlexQuantitationMethod.cpp

```diff
@@ -90,27 +102,23 @@ namespace OpenMS
     defaults_.setValue("reference_channel", "126", "The reference channel (126, 127N, 127C, 128N, 128C, 129N, 129C, 130N, 130C, 131).");
     defaults_.setValidStrings("reference_channel", TMTTenPlexQuantitationMethod::channel_names_);
 
-    //    {0.0, 1.0, 5.9, 0.2},   //114
-    //    {0.0, 2.0, 5.6, 0.1},
-    //    {0.0, 3.0, 4.5, 0.1},
-    //    {0.1, 4.0, 3.5, 0.1}    //117
     defaults_.setValue("correction_matrix", ListUtils::create<String>("0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0,"
-                                                               "0.0/0.0/0.0/0.0"),
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0,"
+                                                                      "0.0/0.0/0.0/0.0"),
                        "Correction matrix for isotope distributions (see documentation); use the following format: <-2Da>/<-1Da>/<+1Da>/<+2Da>; e.g. '0/0.3/4/0', '0.1/0.3/3/0.2'");
```

Comment:
maybe unrelated but the whole "see documentation" is not so great : where actually is the documentation? I would expect this to be the tool docu http://ftp.mi.fu-berlin.de/pub/OpenMS/develop-documentation/html/TOPP_IsobaricAnalyzer.html but there is not much more information there (except "see documentation" again!). So if I am new to OpenMS, how would I ever figure out what values to put there? E.g. it is clear that its -2,-1,+1, +2 but its never specified which channels these actually are -- except in the source code?